### ISSUE TITLE
Bump the API of the CRDs

### ISSUE THREAD

**Issue by cmoulliard** [java-operator-sdk/java-operator-sdk#211]
### Issue

Some of the CRDs part of the samples uses an API which is deprecated since K8s 1.16
```
Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
customresourcedefinition.apiextensions.k8s.io/webservers.sample.javaoperatorsdk created
```

### Proposition

Bump the API of the CRDs of the samples to
```
apiextensions.k8s.io/v1
CustomResourceDefinition
```
Some adaptions will be then needed
```
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
#apiVersion: apiextensions.k8s.io/v1beta1
#kind: CustomResourceDefinition
metadata:
  name: webservers.sample.javaoperatorsdk
spec:
  group: sample.javaoperatorsdk
  versions:
    - name: v1
      # Each version can be enabled/disabled by Served flag.
      served: true
      # One and only one version must be marked as the storage version.
      storage: true
      schema:
        openAPIV3Schema:
          type: object
          properties:
            spec:
              type: object
              properties:
                html:
                  type: string
  scope: Namespaced
  names:
    plural: webservers
    singular: webserver
    kind: WebServer
    shortNames:
      - ws
```

**Comment by adam-sandor** [comment]
Thanks for creating the issue @cmoulliard ! Fixing in progress on https://github.com/ContainerSolutions/java-operator-sdk/tree/crd-v1-upgrade

**Comment by teszes** [comment]
Is this the same as #90 ?

**Comment by teszes** [comment]
Closed as fixed except for the mysql one, issue #90 addresses that
